gk/main.c

```diff
@@ -542,6 +597,84 @@ setup_gk_instance(unsigned int lcore_id, struct gk_config *gk_conf)
 	return ret;
 }
 
+/* TODO Add support for flow entries added from the Grantor server. */
```

Comment:
The way we can specially treat the candidate entries when we are trying to add a policy that came from a Grantor server is to discard the oldest request entry (if it exists at all) when none of the current tests finds an entry to discard. The idea is that a policy decision is more important than a request entry. We can tell which request entry is the oldest looking at field `cap_expire_at`.